### ISSUE TITLE
Windows setting CPU Percents on cpu.TimeStat object 

### ISSUE THREAD

**Issue by dadgar** [shirou/gopsutil#420]
This should probably be a different struct as the other methods will accept it and behave incorrectly. Also it causes upstream bugs on code that relies on it not being a percent.

https://github.com/shirou/gopsutil/blob/master/cpu/cpu_windows.go#L140

**Comment by shirou** [comment]
Sorry, currently I don't have time to investigate this. What struct should we use? Could you send a PR?

**Comment by dadgar** [comment]
The CPU package doesn't really have an appropriate struct. I would at the very minimum add documentation stating the different behavior on the Windows platform.

**Comment by dadgar** [comment]
What makes it even more confusing is that `cpu.Times(false)` is in ticks but `cpu.Times(true)` is in percents.

Ticks: https://github.com/shirou/gopsutil/blob/master/cpu/cpu_windows.go#L52-L76
Percents: https://github.com/shirou/gopsutil/blob/master/cpu/cpu_windows.go#L131

**Comment by marcospedreiro** [comment]
I was taking a look at this today and was doing some experimentation [here](https://github.com/marcospedreiro/gopsutil/commit/45aeaebda5dd016be3983f1bf904b18053fa2deb#diff-0e2a4e756eeab6cdd082772c30523324)

I started using the [WMI Code generator tool](https://www.microsoft.com/en-us/download/details.aspx?id=8572) to see what we could get, and saw that the current class we're querying ([win32_perfformatteddata_counters_processorinformation](http://wutils.com/wmi/root/cimv2/win32_perfformatteddata_counters_processorinformation/)) does return percentage values. 

I used the WMI code generator tool to see if there were any other classes that might get us the CPU time values and I saw [win32_perfrawdata_counters_processorinformation](http://wutils.com/wmi/root/cimv2/win32_perfrawdata_counters_processorinformation/). From the linked documentation, they are stated to return identical values, except the latter has a few extra fields. 

However querying my windows computer for `win32_perfrawdata_counters_processorinformation` returned different values that were not percentages. I did a quick swap out to use this new class and got the following results utilizing the following code: 
```go
package main

import (
	"fmt"

	"github.com/marcospedreiro/gopsutil/cpu"
)

func main() {
	fmt.Println("[windows] cpu.Times(false)")
	t, err := cpu.Times(false)
	if err != nil {
		fmt.Println(err)
		return
	}
	for _, ct := range t {
		fmt.Println(ct)
	}

	fmt.Println("[windows] cpu.Times(true)")
	t, err = cpu.Times(true)
	if err != nil {
		fmt.Println(err)
		return
	}
	for _, ctf := range t {
		fmt.Println(ctf)
	}

	return
}
```

Currently when I run this on master I get the following on my windows computer: 
```
[windows] cpu.Times(false)
{"cpu":"cpu-total","user":68496.5,"system":12999.8,"idle":126170.3,"nice":0.0,"iowait":0.0,"irq":0.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
[windows] cpu.Times(true)
{"cpu":"0,0","user":0.0,"system":0.0,"idle":100.0,"nice":0.0,"iowait":0.0,"irq":0.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
{"cpu":"0,1","user":0.0,"system":12.0,"idle":92.0,"nice":0.0,"iowait":0.0,"irq":0.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
{"cpu":"0,2","user":0.0,"system":0.0,"idle":100.0,"nice":0.0,"iowait":0.0,"irq":0.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
{"cpu":"0,3","user":0.0,"system":6.0,"idle":99.0,"nice":0.0,"iowait":0.0,"irq":0.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
{"cpu":"0,4","user":6.0,"system":0.0,"idle":99.0,"nice":0.0,"iowait":0.0,"irq":0.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
{"cpu":"0,5","user":0.0,"system":6.0,"idle":99.0,"nice":0.0,"iowait":0.0,"irq":0.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
```

With my commit that I linked above I get the following:
```
[windows] cpu.Times(false)
{"cpu":"cpu-total","user":69030.2,"system":13576.2,"idle":155644.0,"nice":0.0,"iowait":0.0,"irq":0.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
[windows] cpu.Times(true)
{"cpu":"0,5","user":111033750000.0,"system":19630000000.0,"idle":266419843750.0,"nice":0.0,"iowait":0.0,"irq":196250000.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
{"cpu":"0,4","user":112372968750.0,"system":21042968750.0,"idle":263667656250.0,"nice":0.0,"iowait":0.0,"irq":167031250.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
{"cpu":"0,3","user":111817812500.0,"system":21075781250.0,"idle":264189843750.0,"nice":0.0,"iowait":0.0,"irq":182343750.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
{"cpu":"0,2","user":121851093750.0,"system":16515312500.0,"idle":258717187500.0,"nice":0.0,"iowait":0.0,"irq":147968750.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
{"cpu":"0,1","user":125105781250.0,"system":22633437500.0,"idle":249344375000.0,"nice":0.0,"iowait":0.0,"irq":199062500.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
{"cpu":"0,0","user":108120156250.0,"system":34864062500.0,"idle":254101875000.0,"nice":0.0,"iowait":0.0,"irq":4299218750.0,"softirq":0.0,"steal":0.0,"guest":0.0,"guestNice":0.0,"stolen":0.0}
```

It appears that the `win32_perfrawdata_counters_processorinformation` may be giving us cpu time data as opposed to percentages, but what I've found in terms of documentation so far has not been helpful, not to mention that we are off by a couple orders of magnitude from the `cpu-total` values.

**Comment by Lomanic** [comment]
Fixed by #611.
